.github/project-labeler.yml

```diff
@@ -7,6 +7,9 @@ apps:
 pie-microsite:
 - 'apps/pie-microsite/**/*'
 
+pie-storybook:
```

Comment:
Small thing, but make sure to update the package root version and CHANGELOG manually, as there are quite a few changes going into the config in the root

Reply:
Done!